src/wifi/index.js

```diff
@@ -4,11 +4,35 @@ require('es6-shim');
 var fs = require('fs');
 var machina = require('machina');
 var spawn = require('child_process').spawn;
+var exec = require('child_process').exec; // Use exec only for little commands
+var schedule = require('node-schedule');
 
-var parser = require('./parseOutput.js').parser;
-var emitter = require('./parseOutput.js').emitter;
+var PacketReader = require('./packet-reader');
 
 var QUERY_TIMEOUT = 10*1000*2;
+var ALL_DAY_MAX_LAST_SEEN = 5 * 60 * 1000; // Precision : Up to +10%
```

Comment:
these values should not be constants but arguments. We should set default values if there are not specified by the invoking program, but 6brain should be able to specifiy these.